cmd/ctl/main.go

```diff
@@ -21,15 +21,22 @@ import (
 	"fmt"
 	"os"
 
+	"k8s.io/klog"
+
 	"github.com/jetstack/cert-manager/pkg/util/cmd"
 )
 
 func main() {
 	stopCh := cmd.SetupSignalHandler()
 	cmd := NewCertManagerCtlCommand(os.Stdin, os.Stdout, os.Stderr, stopCh)
 	cmd.Flags().AddGoFlagSet(flag.CommandLine)
-
 	flag.CommandLine.Parse([]string{})
+	fakefs := flag.NewFlagSet("fake", flag.ExitOnError)
+	klog.InitFlags(fakefs)
+	if err := fakefs.Parse([]string{"-logtostderr=false"}); err != nil {
```

Comment:
I know I added this, but... is it problematic for the `version` and `convert` commands? Don't they utilise klog for logging?

Reply:
I guess it just adds useless flags, I'll move it across